app.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "node-app",
+  "env": {
+    "MONGODB_URI": {
+      "description": "Connection string to connect to MongoDB database",
+      "required": true
+    },
+    "NODE_ENV": {
+      "description": "Node environment",
+      "required": true
+    },
+    "NODE_CONFIGENV": {
+      "description": "Configuration environment",
+      "required": true
+    },
+    "JWT_SECRET_KEY": {
```

Comment:
You don't need this in TODO application I believe since there is no user authentication.

Please remove this and appropriate values in config files.

Reply:
removed.